common/services/app/google-analytics.tsx

```diff
@@ -22,19 +22,20 @@ type Props = {
   };
 };
 
-const togglesGaCanIgnore = ['stagingApi', 'apiToolbar', 'disableRequesting'];
-
-export const Ga4DataLayer: FunctionComponent<Props> = ({ data }) => {
-  // We send toggles as an event parameter to GA4 so we can determine the condition in which a particular event took place.
-  // GA4 now limits event parameter values to 100 characters: https://support.google.com/analytics/answer/9267744?hl=en,
-  // so instead of sending the whole toggles JSON blob we send a concatenated string of only the toggles that are turned on. We also remove toggles we know we don't care about.
-  const toggles = data.toggles
-    ? Object.keys(data.toggles)
-        .filter(toggle => {
-          return !togglesGaCanIgnore.includes(toggle);
-        })
-        .map(toggle => {
-          switch (data.toggles?.[toggle]) {
+// We send toggles as an event parameter to GA4 so we can determine the condition in which a particular event took place.
+// GA4 now limits event parameter values to 100 characters: https://support.google.com/analytics/answer/9267744?hl=en
+// So instead of sending the whole toggles JSON blob we send a concatenated string of only the toggle names (preceeded with a ! if the toggle is false). We also remove toggles we know we don't care about, i.e. those that aren't part of a test
```

Comment:
If I got the code below right, would it be worth rephrasing to; 
`
So instead of sending the whole toggles JSON blob, we only look at the "test" typed toggles and send a concatenated string made of the toggles' name, preceeded with a ! if its value is false.
`
(this also helps confirm I've understood, haha)